models/demos/llama3/lt

```diff
@@ -52,6 +53,204 @@ def ensure_ttsmi_installed():
         sys.exit(1)
 
 
+class OutputEntryList:
+    def __init__(self):
+        self._entries = []
+        # Create logs directory
+        os.makedirs("logs", exist_ok=True)
+        # Load existing state
+        self._load_state()
+
+    def _load_state(self):
+        try:
+            with open("logs/state.json", "r") as f:
+                state = json.load(f)
+                for entry_data in state:
+                    entry = Entry(
+                        entry_data["command_name"],
+                        entry_data["model"],
+                        entry_data["device"],
+                        entry_data["command_input"],
+                    )
+                    # Restore saved attributes
+                    entry.status = (
+                        "Cancelled"
+                        if entry_data["status"]
```

Comment:
Should status be "cancelled" if the status is in this list?

Reply:
Yes - this is to account for cases in which lt was terminated without its usual shutdown (e.g. ird timeout, connection lost etc). Those processes are gone and are not coming back.